diff --git a/node_modules/react-native/scripts/cocoapods/utils.rb b/node_modules/react-native/scripts/cocoapods/utils.rb
index a9cf0fa..0936b4f 100644
--- a/node_modules/react-native/scripts/cocoapods/utils.rb
+++ b/node_modules/react-native/scripts/cocoapods/utils.rb
@@ -448,10 +448,10 @@ class ReactNativePodsUtils
         current = version["str"]
         min_required = Helpers::Constants.min_xcode_version_supported
 
-        if Gem::Version::new(current) < Gem::Version::new(min_required)
-            Pod::UI.puts "React Native requires XCode >= #{min_required}. Found #{current}.".red
-            raise "Please upgrade XCode"
-        end
+        # if Gem::Version::new(current) < Gem::Version::new(min_required)
+        #     Pod::UI.puts "React Native requires XCode >= #{min_required}. Found #{current}.".red
+        #     raise "Please upgrade XCode"
+        # end
     end
 
     def self.add_compiler_flag_to_project(installer, flag, configuration: nil)
